src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,68 @@
+import { FC } from 'react';
+
+export interface IPagination {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<IPagination> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  let totalPages = Math.ceil(total / perPage);
+
+  const changePage = (page: number) => {
+    if (page >= 1 && page <= totalPages) {
+      onPageChange(page);
+    }
+  };
+
+  const renderPageButtons = () => (
+    Array.from({ length: totalPages }, (_, index) => (
+      <li 
+      key={index+1}
+        className={`page-item ${currentPage===index+1?"active":''}`} 
+        onClick={() => changePage(index + 1)}
+        >
+        <a data-cy="pageLink" className="page-link" href={`#${index+1}`}>
+          {index+1}
+        </a>
+      </li>
+    ))
+);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage===1?"disabled":''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
```

Comment:
Using `href="#prev"` might not be the best choice for accessibility. Consider using a button element or a more descriptive `href` value.